docs/modules/extend/examples/pdf-converter-article-title-with-author-and-date.rb

```diff
@@ -0,0 +1,40 @@
+class PDFConverterArticleTitleWithAuthorAndDate < (Asciidoctor::Converter.for 'pdf')
+  register_for 'pdf'
+
+  def ink_general_heading doc, title, opts
+    return super unless opts[:role] == :doctitle # <1>
+    ink_document_title title, opts # <2>
+    ink_document_metainfo doc, opts # <3>
```

Comment:
Should `ink_document_metainfo` also be renamed to something with `details`?

Reply:
That would be great. Perhaps `ink_document_details`.

Reply:
done with 6fd0ed56fd5bc0817c95dbc4a001d141a30d9ed7